Project movies - Moa Hammarbäck

mhammarback opened the pull request:
Link to my deployed project:
https://technigo-movie-project.netlify.app/

My reflections on how this week's project turned out:
I only had time to do the blue level this week but i am happy with the end result. I think React is fun!
If i had more time i would have created an error page and maybe a navbar where you can choose more movies. 

Things I'd like to have clarified or explained in more detail:
nothing special this week!

IngelaL commented on code/src/components/MovieDetail.js:
```diff
@@ -0,0 +1,46 @@
+                <img className="movie-poster" src={`https://image.tmdb.org/t/p/w342${movies.poster_path}`} alt={movies.title} /> 
+          </div>  
+          <div className="movie-detail-text"> 
+            <h1 className="movie-detail-title">{movies.title}
```
I missed the link in the movie title that you showed on your demo, what happened with it?

IngelaL reviewed (approved):
Hi Moa! Your site looks great and the code is very well structured! 👏 I have made some small remarks on some details, such ass adding ";" in some places. If you want to improve the movie detail pages in mobile view you could just do a small adjustment. See what I mean in comment. Really great job Moa!! ✨

mhammarback commented:
Thank you for your code review Ingela! 
I agree, i have changed the movie details page layout so that is a little bit more easy on the eyes :) 
Also changed the mobile view on movie list and decided to display 2 movies/ row instead of all 4! 
The link is now added, i had already done the PR when i found it so that is why it's not here. I just showed it and then i added it now after your review! 

And those ; .. yes consistency is key, need to be more aware of the details!

EmOlsz commented on code/src/components/PopularMovieCard.js:
```diff
@@ -0,0 +1,23 @@
+    
+return (
+  <>
+    <div className="movie-card">
```
If your component returns only one root element, there is no need to wrap it around React Fragment.

In this case WE NEED to use React Fragment :

```jsx
return (
  <div>My element 1</div>
  <div>My element 2</div>
);
```

so, after necessary changes it looks like this :

```jsx
return (
	<>
	  <div>My element 1</div>
	  <div>My element 2</div>
	</>
);
```

But in this case WE DON'T NEED to use React Fragment :

```jsx
return (
	<div>My element 1</div>
);
```

EmOlsz commented on code/src/components/MovieDetail.js:
```diff
@@ -0,0 +1,50 @@
+    <section className="detail-backdrop" style={{  backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) 70%, rgb(0, 0, 0) 100%),  url(https://image.tmdb.org/t/p/w1280${movies.backdrop_path})` }}>
+      <div className="movie-detail-content">
+        <div className="movie-detail-back">
+          <Link to ="/" className="link" exact='true'>
```
There is limited set of props available for `<Link />` predefined component provided by `react-router-dom` package and `exact` is not one of them - basically, it's not doing anything as it's not available for it. It's only available for `<Route />` and `<NavLink />` components.

For further reference check out [https://reactrouter.com/web/api/Link](https://reactrouter.com/web/api/Link)
